### PR TITLE
Support for themes and customizations

### LINKED ISSUE #8: Customization: Themes and Webpacks Support

### PR DESCRIPTION
**Pull request by ruchitpatell**
closes #8 , does not actually add themes, just the support to add them

### CODE REVIEW

**Review by AryanAmin — APPROVED** [review]

**Comment by ruchitpatell** [comment]
test comment

File: breakout.py [hunk 1]
```diff
@@ -1,53 +1,69 @@
 import pygame
 import sys
-from objects.ball import Ball
-from objects.brick import Bricks
-from objects.paddle import Paddle
-from objects.constants import PADDLE_STARTING_POS, SCREEN_WIDTH, SCREEN_HEIGTH
```
Comment (ruchitpatell) [inline breakout.py:6]: test comment

Comment (ruchitpatell) [reply breakout.py:6]: @ruchitpatell

**Comment by ruchitpatell** [comment]
Looks good to me! 🚀

**Pull request merged**